app/reporting-framework/json-reports/lung-cancer-treatment-monthly-summary-base.json

```diff
@@ -14,7 +13,16 @@
             "join": {
                 "type": "inner",
                 "joinCondition": "flct.person_id = patient_name.person_id and (patient_name.voided is null || patient_name.voided = 0)"
-                
+
+            }
+        },
+        {
+            "table": "amrs.person",
+            "alias": "pp",
+            "join": {
+                "type": "inner",
+                "joinCondition": "flct.person_id = pp.person_id and (pp.voided is null || pp.voided = 0)"
```

Comment:
consider using the **OR** instead because this a configurable setting in MySQL. It could be affected my migrations.